### PR TITLE
[Frontend_Web_Development_HTML_CSS_JS] : CSS Classes and IDs

### LINKED ISSUE #951: FRONTEND: HTML classes and ids
<hr>

## Description 📜

I want to add documentation of HTML classes and ids with code snippets.
Assign this issue to me.

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev (HTML, CSS, JS)

<hr>

### PR DESCRIPTION
**Pull request by subhendudash02**
<hr>

## Description 📜

1. Added the documentation of CSS classes and IDs in the form of markdown file.
2. Added the separate sub-folder for screenshots, HTML and CSS files.

Fixes #951 

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.blueHeading {
+    color: #0000FF
+}
+ ```
+
+ <br>
+
+ The output is
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:33]: Make alignment of the output images as 

Output is
`<picture here below>`

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+ The CSS classes can be used repeatedly to any part of the HTML file.
+
+ For example, 
+
+``` HTML
+<h1 class = "redHeading">Red</h1>
+<h2 class = "redHeading">Also Red</h2> 
+```
+
+The output is
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:47]: Make alignment of the output images as 

Output is
`<picture here below>`

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+In CSS file
+
+``` CSS
+#myHeading {
+    color: #008000;
+    font-family: "Comic Sans MS";
+}
+```
+
+The output is
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:75]: Make alignment of the output images as 

Output is
`<picture here below>`

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+In JavaScript 
+
+``` JavaScript
+"use strict";
+let x = document.getElementById("main");
+x.innerHTML = "Embedded by JavaScript";
+```
+
+The output is
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:98]: Make alignment of the output images as 

Output is
`<picture here below>`

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (code-reaper08) [inline .vscode/settings.json:3]: You don't need this file `.vscode/settings.json`

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
I've left a review, look into it and do the needful.

**Comment by subhendudash02** [comment]
@code-reaper08 I have modified the file as suggested.
Thank you!

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Hey I was meaning about the alignment of the output image.

**What you have now:**

Output_alignment

**What would be better**

It would be better if the screenshot is below the word "Output is".

Output is

Expected

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+On inspecting the element in any suitable browser,
+![Inspect Element](Images/Inspect_Element_CSS_IDs.png)
+
+The `<h2>` content is not visible on inspecting.
+<br>
+
+Therefore, IDs are supposed to be used once.
+
+<hr><br><hr>
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:111]: You can remove this too.

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+<br><hr><br>
+
+# References
+1. https://www.w3schools.com/html/html_id.asp
+2. https://www.w3schools.com/cssref/sel_class.asp
+3. https://www.geeksforgeeks.org/difference-between-an-id-and-class-in-html/
+4. https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Type_Class_and_ID_Selectors
+
+<br><hr><br>
```
Comment (code-reaper08) [inline Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:126]: And this one too.

**Review by code-reaper08 — COMMENTED** [review]
Do change these :-)

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md (hunk 7 above)
Comment (subhendudash02) [reply Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:126]: Oh you mean br and hr tag?

**Comment by subhendudash02** [comment]
image
I don't know, but the image is showing below the content in my text-editor (VS-Code).

**Comment by code-reaper08** [comment]
@2024-SANDHYA have a look here.

File: Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md (hunk 7 above)
Comment (code-reaper08) [reply Web_Development/FrontEnd/CSS_Classes_And_IDs/README.md:126]: Yes

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Sure @code-reaper08 . @subhendudash02  Everything looks fine, just work a bit on content alignment. Like here you can see the note thing should be in one line only and so on..

image

**Comment by subhendudash02** [comment]
Okay, noted @2024-SANDHYA and @code-reaper08 , the alignment is the big issue here, I will work on this problem tonight.
Thank you so much for the review.🙌

**Comment by subhendudash02** [comment]
I have worked on the alignment of the document @code-reaper08, @2024-SANDHYA. Kindly check.

**Review by 2024-SANDHYA — APPROVED** [review]
Looks okayish! @code-reaper08 have a look

**Comment by Alankrita9** [comment]
html class and id selectors has already been merged

**Comment by subhendudash02** [comment]
> html class and id selectors has already been merged

I know that, I am not demanding to merge this PR, I was demanding to just look for the mistakes..... I am aware of the fact it is already merged.

**Review by code-reaper08 — COMMENTED** [review]
Looks nice @subhendudash02 🆒

**Pull request closed without merging**